catalog/meta.go

```diff
@@ -2,6 +2,9 @@ package catalog
 
 // Meta structure describes engine metadata used in linterhub
 type Meta struct {
+	//Recommendation for install, if this engine can be useful;
+	// Used by catalog --project strategy
+	Recommendation string `json:"Recommendation,omitempty"`
```

Comment:
If you need to extend object - please use inheritance, the original `Meta` schema do not have such property.